web-client/src/modules/myRequests/routes.tsx

```diff
@@ -8,7 +8,9 @@ const FindRequestsContainer = lazy(() =>
   import('./containers/FindRequestsAlgoliaContainer'),
```

Comment:
Refactored MyPostsTabPage to be in separate dir from requests or offers now that there are two dirs.